util/passenger/passenger.go

```diff
@@ -35,13 +77,15 @@ func (e ErrTokenNotMatchingUser) Error() string {
 	return fmt.Sprintf("passenger: parent of token %+v does not match with actual user %+v", e.Parent, e.Actual)
 }
 
+func init() {
+	passengerKey = key(mathrand.Int63())
```

Comment:
The `passengerKey` is not used anywhere. Will it be used somewhere in the future?